code/modules/spells/ability_types/realized.dm

```diff
@@ -864,3 +865,197 @@
 	H.adjust_attribute_buff(PRUDENCE_ATTRIBUTE, -10)
 	H.adjust_attribute_buff(TEMPERANCE_ATTRIBUTE, -10)
 	H.adjust_attribute_buff(JUSTICE_ATTRIBUTE, 10)
+
+/* Flesh Idol - Repentance */
+/obj/effect/proc_holder/ability/prayer
+	name = "Prayer"
+	desc = "An ability that does causes you to start praying reducing damage taken by 50% but removing your ability to move and lowers justice by 80. \
+	When you finish praying everyone in a big area gets a 25% damage boost and gets healed."
+	action_icon_state = "flesh0"
+	base_icon_state = "flesh"
+	cooldown_time = 30 SECONDS
+
+	var/damage_amount = 300 // Amount of explosion damage
+	var/explosion_range = 15
+
+/obj/effect/proc_holder/ability/prayer/Perform(target, mob/living/carbon/human/user)
+	ADD_TRAIT(user, TRAIT_IMMOBILIZED, type)
+	user.apply_status_effect(/datum/status_effect/flesh1)
+	cooldown = world.time + (15 SECONDS)
+	to_chat(user, "<span class='userdanger'>You start praying...</span>")
+	if(!do_after(user, 15 SECONDS))
+		REMOVE_TRAIT(user, TRAIT_IMMOBILIZED, type)
+		return
+	REMOVE_TRAIT(user, TRAIT_IMMOBILIZED, type)
+	for(var/mob/living/carbon/human/H in GLOB.player_list)
+		if(!user.faction_check_mob(H, FALSE))
+			continue
+		if(H.stat == DEAD)
+			continue
+		playsound(H, 'sound/abnormalities/onesin/bless.ogg', 100, FALSE, 12)
+		to_chat(H, "<span class='nicegreen'>[user]'s prayer was heard!</span>")
+		H.adjustBruteLoss(-100)
+		H.adjustSanityLoss(-100)
+		H.apply_status_effect(/datum/status_effect/flesh2)
+		new /obj/effect/temp_visual/healing(get_turf(H))
+	return ..()
```

Comment:
Overall way too much ESPECIALLY on a 30 second cooldown.
1. Turns you into a super tank, with a minimum resistance of 70%.
2. Heals EVERYONE regardless of location 100 HP and SP
3. Gives EVERYONE regardless of location +25% damage.

Sitting still is not a punishment if you become super beefy for it. Stack this with KoD blessing and you sit at lowest resistance being 85% and your highest being 97.5%.

Reply:
meant for this to have a cooldown of 180 seconds